projects/robot_arm/requirements.txt

```diff
@@ -20,5 +20,6 @@
 #
 # ------------------------------------------------------------------------------
 # Requirements specific to learning by disagreement project
+opencv-python
```

Comment:
For some reason it is not being installed by real_robots, although it is a requirement on that lib.